src/data_proc.py

```diff
@@ -9,7 +9,7 @@ class DataProc(torch.utils.data.Dataset):
 
     def __init__(self, args):
         self.args = args
-        self.data_dict = pickle.load(open('%s/data.pickle'%(args.dataset),'rb'))
+        self.data_dict = pickle.load(open('%s.pickle'%(args.dataset),'rb'))
```

Comment:
lets keep it as it was before to provide the root path to the dataset and pass additional argument to data_proc to determine  which split to load; could be something like this`pickle.load(open('%s/data_%s.pickle'%(args.dataset, split),'rb'))`

Reply:
Good idea, on it